server/README.md

```diff
@@ -1 +1,67 @@
 # Pailot Backend
+
+This is a Node.js application built with TypeScript
+
+# Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+# Requirement
+
+To run this application, you need to have the following software installed on your system:
+
+Node.js (version 12 or higher)
+npm/yarn
+
+# Installation
+
+Clone the repository:
+
+```bash 
+git clone https://github.com/pi-apps/pailot-for-pi.git
+
+```
+
+Navigate to the project directory:
+
+```TypeScript
```

Comment:
This should not be typescript. It is not a typescript code.
It is a shell or bash command.